### DISCUSSION TITLE
What schema version are folks using? v1, v2, v3.1, latest as updated?

### DISCUSSION THREAD

**Discussion by Data-san** [CMSgov/price-transparency-guide#210] [Q&A] [answered]
Appreciate any insights on how respective teams are handling schema updates. Perhaps there could be more. Thanks.

**Comment by ericsilvertx** [comment]
I will be adjusting my code to fit the latest schema.  Evidently as we get closer to the January 2021 deadline i might change my opinion

**Comment by shaselton-usds** [comment, marked as answer]
Version 1.0 of the schema is required to meet compliance. Version 1.0 was finalized into the master branch on March 1st, 2022
